app/scripts/modules/core/src/application/nav/NavSection.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import {
+  mockLoadBalancerDataSourceConfig,
+  mockPipelineDataSourceConfig,
+  mockServerGroupDataSourceConfig,
+} from '@spinnaker/mocks';
+import { Application } from '../../application';
+import { NavSection } from './NavSection';
+
+describe('NavCategory', () => {
+  it('should render multiple categories', () => {
+    const app = new Application('testapp', null, [
```

Comment:
I think we usually use [this builder method](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/application/applicationModel.builder.ts#L8-L11) in tests? I haven't used it personally before so don't take my word for it.

Reply:
Yes we do. It is just a simple wrapper around this function but I can swap it out.